File: Iniciante/1036.c
#include <stdio.h>
#include <math.h>

int main(){

    double a, b, c, delta;

    scanf("%lf %lf %lf", &a, &b, &c);
    
    delta = pow(b, 2) -4.0*a*c;

    if(delta < 0 || a == 0){
        printf("Impossivel calcular\n");
    }else{
        printf("R1 = %.5f\nR2 = %.5f\n", (-b + sqrt(delta)) / (2.0 * a), (-b - sqrt(delta)) / (2.0 * a));
    }

    return 0;
}

File: Iniciante/1036.php
<?php

$L1 = explode(" ", readline());

$a = $L1[0];
$b = $L1[1];
$c = $L1[2];

$delta = $b**2 -4*$a*$c;

if($delta<0 || $a==0){
    echo "Impossivel calcular\n";
}else{
    $x1 = (-$b + $delta**0.5)/(2*$a);
    $x2 = (-$b - $delta**0.5)/(2*$a);
    echo "R1 = ".number_format($x1, 5, '.', '')."\n";
    echo "R2 = ".number_format($x2, 5, '.', '')."\n";
}

?>